docs/src/admin-manual/configuration.md

```diff
@@ -459,15 +459,19 @@ xsdPath = "/home/enduro/premis.xsd"
 ### Watched location configuration
 
 These configuration settings, when enabled, allow Enduro to initiate SIP ingest
-from a watched location. The configured watched location can either be a local
-filesystem directory or an object store bucket (such as one provided by MinIO,
-S3, or Azure). Once configured, any time a new zipped package is added to the
-location, Enduro's watcher (in this case, [Redis]) will see it and automatically
-initiate an ingest workflow. For more information, see:
+from a watched location. The configured watched location can be an object store
+bucket such as one provided by MinIO, S3, or Azure. Once configured, any time a
+new zipped package is added to the location, it will be configured to publish
```

Comment:
Maybe "It should be configured to publish an event to Enduro's message queue."?  

We have never configured S3 or Azure to publish events to Enduro's Redis queue, so using those two object stores as a watched directory is purely theoretical at this point. A system administrator would have to figure out how to set up S3 or Azure publish Redis events on their own.

Reply:
Ok, here is how I have tried to revised this section: 

> Once this section is configured, the chosen watched location should then be configured to publish an event to Enduro's message queue (in this case, [Redis], listening for events in the queue defined by the `redisList` parameter below) any time a new zipped package is added to the watched location. Enduro's internal watcher will watch for new deposit events at the configured `redisAddress` and queue, and will trigger the ingest workflow when it detects a SIP deposit in the configured watched location.